README.capture-agent-manager.md

```diff
@@ -170,3 +170,45 @@ switching from primary to secondary live stream..
     * redundancy live `redunlive` to toggle from primary to secondary
       live streams (when there is a problem with the primary live)
 
+
+
+
+# Enabling Capture Agent Time Drift Monitoring
+
+The utility node collect time from each capture agent and calculate the
+difference (with its local system time, for now good enough) and post the
+difference as a cloudwatch metric.
+
+To do this add the recipe in the setup phase of the utility node:
+```
+    mh-opsworks-recipes::install-capture-agent-timedrift-metric
```

Comment:
i think i want this metric to be installed very deliberately, because if it might mess up cloudwatch if 2 utility nodes (in different clusters) start to send the same metrics to the test account...